File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-8.js
class Solution {
    /**
     * @param {number[]} arr
     * @return {number[]}
     */
    replaceElements(arr) {
        //arr[i] replace with -> max value on the right subarray
        for(let i = 0; i < arr.length; i++){
            let maxRightNumber = -1;
            for(let k = i + 1; k < arr.length; k++){
                maxRightNumber = Math.max(arr[k], maxRightNumber);
            }
            arr[i] = maxRightNumber;
        }

        return arr;
    }
}

File: Data Structures & Algorithms/is-subsequence/submission-8.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        t_p = 0
        s_p = 0
        
        while s_p < len(s) and t_p < len(t):
            if s[s_p] == t[t_p]:
                s_p += 1
            t_p += 1

        return s_p == len(s)